src/utils/page-functions.js

```diff
@@ -0,0 +1,202 @@
+import { createHash } from 'crypto'
+import path from 'path'
+import webpack from 'webpack'
+import { camelCase } from '../utils'
+import {
+  readFile,
+  exists,
+  stats,
+  requireResolve
+} from './fs'
+
+let packageName
+let nodeModulesPath
+
+/* export async function listPageFunctionSources(fnPath) {
+  if (!fnPath) {
+    fnPath = path.join(__dirname, 'page-functions')
+  }
+
+  const files = await readDir(fnPath, { encoding: 'utf8' })
+  return files.filter(file => file.endsWith('.js'))
+} */
+
+export function createPageFunctions(page, sourceFiles) {
```

Comment:
We may also support source files as a directory (and internally glob) to have a nuxt like experience

Reply:
:+1: 

that was actually the (sort of) task  of the commented out listPageFunctionSources function. But I thought that might be more in place in nuxt/test-utils instead of here. But probably would be ok here as well.